src/ui/input/input_wrap.jsx

```diff
@@ -3,12 +3,16 @@ import React from 'react';
 
 export default class InputWrap extends React.Component {
   render() {
-    const { after, focused, invalidHint, isValid, name, icon } = this.props;
+    const { after, focused, invalidHint, isValid, name, icon, className } = this.props;
     let blockClassName = `auth0-lock-input-block auth0-lock-input-${name}`;
     if (!isValid) {
       blockClassName += ' auth0-lock-error';
     }
 
+    if (className) {
```

Comment:
Needed a way to maintain the CSS for the CheckBoxInput so added this `className` proper to allow that